app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/organisation/GrsController.scala

```diff
@@ -109,7 +109,7 @@ class GrsController @Inject() (
                   else
                     Redirect(commonRoutes.NotableErrorController.duplicateRegistration())
                 case UNSUPPORTED_ORGANISATION =>
-                  Redirect(orgRoutes.OrganisationTypeNotSupportedController.onPageLoad())
+                  Redirect(orgRoutes.RegisterAsOtherOrganisationController.onPageLoad())
```

Comment:
I'm thinking the GRS_FAILED status should cover GRS problems. There are code paths where it could end up here for those not listed but that "shouldn't" happen. If it does then it should be treated as a bug at that point.